exporter/local/export.go

```diff
@@ -102,20 +104,44 @@ func (e *localExporterInstance) Export(ctx context.Context, inp *exporter.Source
 				defer cleanup()
 			}
 
+			if !e.opts.Split {
+				err = outputFS.Walk(ctx, func(p string, fi os.FileInfo, err error) error {
+					if fi.IsDir() {
+						return nil
+					}
+					if err != nil && !errors.Is(err, os.ErrNotExist) {
+						return err
+					}
+					st, ok := fi.Sys().(*fstypes.Stat)
```

Comment:
Does this work with attestations? I don't think these files have this type. Why is the type check needed and just `p` is not used?

Reply:
Good point, I didn't try with attestation as the implementation was done before. I will take a look.